common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -789,7 +789,11 @@ function (Sjson, AsyncProcess) {
             }
 
             hideSubtitlesEl
-                .attr('title', text)
+                .attr('title', gettext(text))
+                .removeClass('show_transcript')
+                .removeClass('hide_transcript')
```

Comment:
Why do we need to remove these class names? Does it really has these class names?